ibm_catalog.json

```diff
@@ -158,6 +201,287 @@
+                },
+                {
+                  "title": "Creates a queue manager",
+                  "description": "Creates a queue manager"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "IBM MQ on Cloud",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-mq-cloud/main/reference-architecture/deployable-architecture-mqcloud.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture supports creating and configuring MQ on Cloud resources"
+                }
+              ]
+            },
+            "dependencies": [
+              {
+                "name": "testing-deploy-arch-ibm-secrets-manager",
+                "id": "5f149ae6-d781-460a-977d-c8c4171ad0fa-global",
+                "flavors": [
+                  "standard"
+                ],
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "version": "^1.2.1",
+                "optional": true,
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "resource_group_name",
+                    "version_input": "existing_resource_group_name"
+                  },
+                  {
+                    "value": true,
+                    "dependency_input": "use_existing_resource_group"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "region",
+                    "version_input": "region"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "prefix",
+                    "version_input": "prefix"
+                  },
+                  {
+                    "dependency_output": "secrets_manager_crn",
+                    "version_input": "existing_secrets_manager_crn"
+                  }
+                ],
+                "ignore_auto_referencing": [
+                  "*"
+                ]
+              },
+              {
+                "name": "testing-deploy-arch-ibm-observability",
+                "id": "daf2cf91-2384-4205-b7f4-621299a4896d-global",
+                "flavors": [
+                  "standard"
+                ],
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "optional": true,
+                "version": "^1.2.2",
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "resource_group_name",
```

Comment:
No need to map resource group for observability, it already defines its own mapping with account base in the observability DA, so you get it free here.